api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -1547,6 +1547,97 @@ message AccessRequestCreate {
   ];
 }
 
+// AccessRequestExpire is emitted when access request has expired.
+message AccessRequestExpire {
+  // Metadata is a common event metadata
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // User is a common user event metadata
+  UserMetadata User = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // ResourceMetadata is a common resource event metadata
+  ResourceMetadata Resource = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // Roles is a list of roles for the user.
+  repeated string Roles = 4 [(gogoproto.jsontag) = "roles,omitempty"];
+
+  // RequestID is access request ID
+  string RequestID = 5 [(gogoproto.jsontag) = "id"];
+
+  // RequestState is access request state (in the access_request.review variant of
+  // the event this represents the post-review state of the request).
+  string RequestState = 6 [(gogoproto.jsontag) = "state"];
+
+  // Delegator is used by teleport plugins to indicate the identity
+  // which caused them to update state.
+  string Delegator = 7 [(gogoproto.jsontag) = "delegator,omitempty"];
+
+  // Reason is an optional description of why the request is being
+  // created or updated.
+  string Reason = 8 [(gogoproto.jsontag) = "reason,omitempty"];
```

Comment:
Do we need to be duplicating all these fields from the request create event?

The delete event include only request ID - should we also only include request ID here (so it can be traced back to the create event) and the expiry time?